bin/cleaver

```diff
@@ -84,6 +84,25 @@ program
       });
     });
 
+    // Watch theme files on local repository if included via command prompt
+    if(options.theme) {
+      if(fs.existsSync(options.theme)) {
+        var files = fs.readdirSync(options.theme);
+        files.forEach (function(theme_file) {
+          var file_extension = theme_file.substr(theme_file.lastIndexOf('.') + 1);
```

Comment:
probably, better to use [path.extname](https://nodejs.org/api/path.html#path_path_extname_path)